src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -229,22 +214,18 @@ def save_specification(self, specification: ForecastSpecification):
     def save_raw_covariates(self, covariates: pd.DataFrame, scenario: str, draw_id: int):
         io.dump(covariates, self.forecast_root.raw_covariates(scenario=scenario, draw_id=draw_id))
 
-    def load_raw_covariates(self, scenario: str, draw_id: int):
-        covariates = io.load(self.forecast_root.raw_covariates(scenario=scenario, draw_id=draw_id))
-        covariates['date'] = pd.to_datetime(covariates['date'])
-        return covariates
-
     def load_beta_params(self, draw_id: int) -> Dict[str, float]:
         df = io.load(self.regression_root.parameters(draw_id=draw_id))
         return df.set_index('params')['values'].to_dict()
 
+    def save_ode_params(self, ode_params: pd.DataFrame, scenario: str, draw_id: int):
```

Comment:
Saving beta/theta/etc. separately from the compartments to reduce load time in postprocessing. The task producing final betas was 2x longer than the next longest task.